sopel/bot.py

```diff
@@ -372,6 +421,12 @@ def action(self, text, dest):
     def reply(self, text, dest, reply_to, notice=False):
         """Prepend ``reply_to`` to ``text``, and send as a PRIVMSG to ``dest``.
 
+        :param str text: the text of the reply
+        :param str dest: the destination of the reply
+        :param str reply_to: the nickname that the reply will be prepended with
+        :param bool notice: whether to send the reply as a NOTICE or not,
+                            defaults to `False`
```

Comment:
We need the double backticks, actually. Single backticks produce the wrong output (usually italics).

```suggestion
                            defaults to ``False``
```